### ISSUE TITLE
Modified recurrence summary is lost

### ISSUE THREAD

**Issue by amelchio** [iCalcreator/iCalcreator#66]
The below test program prints
```
2018-10-23 12:00:00 Europe/Copenhagen Original
2018-10-24 14:00:00 Europe/Copenhagen Original
2018-10-25 12:00:00 Europe/Copenhagen Original
```

while I would expect the second line to be "Changed", not "Original". The modified time works but the change to the summary is lost.

```php
<?php

$ical = <<<EOF
BEGIN:VCALENDAR
PRODID:-//Google Inc//Google Calendar 70.9054//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-TIMEZONE:Europe/Copenhagen
BEGIN:VTIMEZONE
TZID:Europe/Copenhagen
X-LIC-LOCATION:Europe/Copenhagen
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
DTSTART;TZID=Europe/Copenhagen:20181024T140000
DTEND;TZID=Europe/Copenhagen:20181024T150000
DTSTAMP:20181022T205931Z
UID:45funnystpjohnny1isjcd03vp@google.com
RECURRENCE-ID;TZID=Europe/Copenhagen:20181024T120000
CREATED:20181022T172215Z
DESCRIPTION:
LAST-MODIFIED:20181022T172249Z
LOCATION:
SEQUENCE:1
STATUS:CONFIRMED
SUMMARY:Changed
TRANSP:OPAQUE
END:VEVENT
BEGIN:VEVENT
DTSTART;TZID=Europe/Copenhagen:20181021T120000
DTEND;TZID=Europe/Copenhagen:20181021T130000
RRULE:FREQ=DAILY
EXDATE;TZID=Europe/Copenhagen:20181022T120000
DTSTAMP:20181022T205931Z
UID:45funnystpjohnny1isjcd03vp@google.com
CREATED:20181022T172215Z
DESCRIPTION:
LAST-MODIFIED:20181022T172215Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Original
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
EOF;

require_once('iCalcreator-2.24/autoload.php');

$cal = new kigkonsult\iCalcreator\vcalendar();
$cal->parse($ical);

$events_arr = $cal->selectComponents(2018, 10, 23, 2018, 10, 25);
foreach( $events_arr as $year => $year_arr ) {
    foreach( $year_arr as $month => $month_arr ) {
        foreach( $month_arr as $day => $day_arr ) {
            foreach( $day_arr as $event ) {
                $dtstart = $event->getProperty("x-current-dtstart");
                $datetime = $dtstart[1];
                print "$datetime " . $event->getProperty("summary") . "\n";
            }
        }
    }
}
```

**Comment by iCalcreator** [comment]
Looking into the issue!

**Comment by iCalcreator** [comment]
Fixed in the 2.24.2 release!

**Comment by amelchio** [comment]
Thank you!
